ecosystem/indexer-grpc/indexer-grpc-integration-test-transaction-generator/src/lib.rs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+pub mod capture_transactions;
+pub mod move_executor;
+pub mod transaction_generator;
+
+// Move fixtures directory name.
+pub const MOVE_FIXTURES_DIR: &str = "move_fixtures";
```

Comment:
Given they can also be raw txns in JSON format, should it be `txn_fixtures`? Or maybe this dir is only for Move code and nothing else?